Snippets/Snippets_4/Scheduling/Scheduling.cs

```diff
@@ -14,7 +14,7 @@ public void ScheduleTask()
 
         // Name a schedule task and invoke it every 5 minutes
         Schedule.Every(TimeSpan.FromMinutes(5))
-            .Action("Task name", SomeCustomMethod);
+            .Action("MyCustomTask", SomeCustomMethod);
```

Comment:
I wanted the saga name to correlate with the task name. In the last section, I'll make this more obvious. Thanks.